File: round 1/Beast/problem3.java
#include<stdio.h>
int main()
{
	int n;
	scanf("%d",&n);
	long long int sum=1;
	for(int i=1;i<=n;i++)
	{
		if(i==n)
		{
		 printf("%lld",(sum*sum));	
		}
		else
		printf("%lld\n",(sum*sum));
		sum=sum*10+1;
	}
}

File: q/c/submissions/B_D/problem3.c
#include<stdio.h>
int main()
{
int n,i,j,k;
scanf("%d",&n);
if(n<10)
{
for(i=1;i<=n;i++)
{
for(j=1;j<=i;j++)
{
 printf("%d",j);
}
for(k=(i-1);k>=1;k--)
{
printf("%d",k);
}
printf("\n");
}
}
}